logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/publicuser/search/SearchKnownUsersUseCase.kt

```diff
@@ -60,16 +60,18 @@ internal class SearchKnownUsersUseCaseImpl(
         val sanitizedSearchQuery = searchQuery.lowercase()
         return if (isUserLookingForHandle(sanitizedSearchQuery)) {
             searchUserRepository.searchKnownUsersByHandle(
-                handle = sanitizedSearchQuery.removePrefix("@"),
+                handle = sanitizedSearchQuery.substringAfter("@").substringBeforeLast("@"),
```

Comment:
suggestion: maybe we could extract it and create a dedicated function to be able to reuse it if needed? 🤔